p4_symbolic/ir/ir.proto

```diff
@@ -75,11 +84,11 @@ message Action {
 
 // Action implementation details.
 message ActionImplementation {
-  // Type of all variables used in action_body.
-  // Maps a variable name to its bitwidth.
-  map<string, int32> variables = 1;
+  // Type of all action parameters.
+  // Maps a parameter name to its bitwidth.
+  map<string, int32> parameters = 2;
```

Comment:
Consider renaming to parameters_and_bitwidths

Reply:
how about parameter_to_bitwidth